### PR TITLE
[Bug Fix] Fixed message on promote/demote permissions check.

### PR DESCRIPTION
**Pull request by Valorith**
Old message when you attempted to promote or demote a player and did not have permissions to do so: "You dont have permission to invite."

New promote rejection message: "You dont have permission to promote."
New demote rejection message: "You dont have permission to demote."

### CODE REVIEW

File: zone/client_packet.cpp [hunk 1]
```diff
@@ -7599,7 +7599,7 @@ void Client::Handle_OP_GuildDemote(const EQApplicationPacket *app)
 	if (!IsInAGuild())
 		Message(0, "Error: You arent in a guild!");
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:7600]: `aren't`

File: zone/client_packet.cpp [hunk 2]
```diff
@@ -7599,7 +7599,7 @@ void Client::Handle_OP_GuildDemote(const EQApplicationPacket *app)
...
 	else if (!guild_mgr.CheckPermission(GuildID(), GuildRank(), GUILD_DEMOTE))
-		Message(0, "You dont have permission to invite.");
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:7602]: `don't`

File: zone/client_packet.cpp [hunk 3]
```diff
@@ -8045,7 +8045,7 @@ void Client::Handle_OP_GuildPromote(const EQApplicationPacket *app)
 	if (!IsInAGuild())
 		Message(0, "Error: You arent in a guild!");
 	else if (!guild_mgr.CheckPermission(GuildID(), GuildRank(), GUILD_PROMOTE))
-		Message(0, "You dont have permission to invite.");
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:8048]: `don't`

File: zone/client_packet.cpp [hunk 4]
```diff
@@ -8045,7 +8045,7 @@ void Client::Handle_OP_GuildPromote(const EQApplicationPacket *app)
 	if (!IsInAGuild())
 		Message(0, "Error: You arent in a guild!");
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:8046]: `aren't`

**Comment by Kinglykrab** [comment]
`Message(Chat::White` instead of `Message(0` where you're changing stuff.

File: zone/client_packet.cpp (hunk 1 above)
Comment (Valorith) [reply zone/client_packet.cpp]: Fixed

File: zone/client_packet.cpp (hunk 2 above)
Comment (Valorith) [reply zone/client_packet.cpp:7602]: Fixed

File: zone/client_packet.cpp (hunk 3 above)
Comment (Valorith) [reply zone/client_packet.cpp:8048]: Fixed

File: zone/client_packet.cpp (hunk 4 above)
Comment (Valorith) [reply zone/client_packet.cpp]: Fixed

File: zone/client_packet.cpp [hunk 5]
```diff
@@ -7919,9 +7919,9 @@ void Client::Handle_OP_GuildLeader(const EQApplicationPacket *app)
 	app->pBuffer[app->size - 1] = 0;
 	GuildMakeLeader* gml = (GuildMakeLeader*)app->pBuffer;
 	if (!IsInAGuild())
-		Message(0, "Error: You arent in a guild!");
+		Message(0, "Error: You aren't in a guild!");
 	else if (GuildRank() != GUILD_LEADER)
-		Message(0, "Error: You arent the guild leader!");
+		Message(0, "Error: You aren't the guild leader!");
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:7924]: `Chat::Red`

File: zone/client_packet.cpp [hunk 6]
```diff
@@ -8043,9 +8043,9 @@ void Client::Handle_OP_GuildPromote(const EQApplicationPacket *app)
 	}
 
 	if (!IsInAGuild())
-		Message(0, "Error: You arent in a guild!");
+		Message(0, "Error: You aren't in a guild!");
 	else if (!guild_mgr.CheckPermission(GuildID(), GuildRank(), GUILD_PROMOTE))
-		Message(0, "You dont have permission to promote.");
+		Message(Chat::Red, "You don't have permission to promote.");
 	else if (!worldserver.Connected())
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:8049]: `Chat::Red`

File: zone/client_packet.cpp [hunk 7]
```diff
@@ -8131,13 +8131,13 @@ void Client::Handle_OP_GuildRemove(const EQApplicationPacket *app)
 	}
 	GuildCommand_Struct* gc = (GuildCommand_Struct*)app->pBuffer;
 	if (!IsInAGuild())
-		Message(0, "Error: You arent in a guild!");
+		Message(0, "Error: You aren't in a guild!");
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:8134]: `Chat::Red`

**Comment by Valorith** [comment]
> `Message(Chat::White` instead of `Message(0` where you're changing stuff.

Changed all of them to use these constants. Changed them to Chat::Red to be more consistent with other similar error colors.

File: zone/client_packet.cpp (hunk 7 above)
Comment (Valorith) [reply zone/client_packet.cpp:8134]: Fixed

File: zone/client_packet.cpp (hunk 6 above)
Comment (Valorith) [reply zone/client_packet.cpp:8049]: Fixed

File: zone/client_packet.cpp (hunk 5 above)
Comment (Valorith) [reply zone/client_packet.cpp:7924]: Fixed

Comment (Valorith) [reply zone/client_packet.cpp:7924]: Fixed

**Review by Kinglykrab — APPROVED** [review]

**Pull request merged**